docs/demo.mdx

```diff
@@ -0,0 +1,370 @@
+
+## Paragraphs Section:
+
+Dummy double-spaced paragraph example, consectetur adipiscing elit. Curabitur euismod,
+
+arcu eu commodo tincidunt, est eros auctor augue, eget bibendum nibh
+
+tellus a mi. Integer blandit, enim quis malesuada fringilla, elit augue
+
+pellentesque justo, eget euismod elit sapien id magna. Sed molestie
+
+ultrices metus, a congue erat.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel **tortor**, bibendum id _pellentesque_ ullamcorper, \*congue** metus. In hac habitasse platea dictumst. Sed auctor, **nunc** eu _pellentesque_ congue, ~~augue est malesuada massa~~, in congue ipsum **mauris\*\* eget `massa` finibus.
+Fusce ~~elementum~~ velit vel **felis** congue, eu _feugiat_ nibh facilisis. Praesent mollis `dolor` id eros [ultrices](#) euismod. Sed ~~euismod~~ posuere **mauris** velit, quis _sodales_ mauris feugiat non. `Consectetur` a ~~ligula~~, ut congue **lacinia**, faucibus _lorem_.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur euismod, arcu eu commodo tincidunt, est eros auctor augue, eget bibendum nibh tellus a mi. Integer blandit, enim quis malesuada fringilla, elit augue pellentesque justo, eget euismod elit sapien id magna. Sed molestie ultrices metus, a congue erat.
+
+#### _BELOW is an example of a single line with bold text and a code span with a link and blue link color and also another link:_
+
+Consectetur elit curabitur **required** by Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur euismod.
+[`dummysilutus`](https://dummyurlpath/DummyAppleSimulatorPathDummy) is via [Dummybrew](https://Dummypath.dummy):
+
+#### _BELOW is a single line with code spans bullet list:_
+
+Sed ut perspiciatis unde omnis iste natus error sit, you still have some work to do with them:
+
+- `dummy.config.js` – Dummy config file;
+- `dummy/dummy.config.js` – Dummy configuration;
+- `dummy/dummy.test.js` – Dummy first test.
+
+#### BELOW is an example of a text line and spacing between an unordered list and code spans:
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur euismod, arcu eu commodo tincidunt, est eros:
+
+- **Dummy scripts:**
+  - `dummy.config.js`
+  - `dummy.config.js`
+- **Dummy test code:**
+  - `some/dummy/url/<your.package>/DummyTest.dummy`
+- **Dummy Manifests:**
+  - `dummy/dummySRC/DummyManifest.xml`
+  - `dummyandroid/dummypath/res/xml/network_security_dummy.dummyxml`
+
+## Admonitions Section:
+
+:::caution
+
+Rhoncus magna eget, pellentesque auguerhoncus magna eget, pellentesque augue **Android** due to DUMMY issues:
+
+- [rhoncus magna eget, pellentesque auguerhoncus magna eget](https://some/url/path)
+- [Curabitur euismod, arcu eu commodo tincidunt, est eros auctor augue](https://some/url/path)
+- [pellentesque auguerhoncus magna eget](ttps://some/url/path)
```

Comment:
I don't advise using incorrect URLs, first. E.g. you have here `ttps:` - missing h letter.

Second, `(https://some/url/path)` does not look good either. Use `#` fallback, e.g. `[Some link](#)` - this should be perfectly enough.

Reply:
Thanks...updated all links!